src/Translator/translations/sk.php

```diff
@@ -0,0 +1,188 @@
+        2 {utorka}
+        3 {stredy}
+        4 {štvrtka}
+        5 {piatka}
+        6 {soboty}
+        7 {nedele}
+        other {{dayNumber} - unknown}
+      }}
+      range {{dayNumber, select,
+        1 {pondelka}
+        2 {utorka}
+        3 {stredy}
+        4 {štvrtka}
+        5 {piatka}
+        6 {soboty}
+        7 {nedele}
+        other {{dayNumber} - unknown}
+      }}
+      other {{dayNumber, select,
+        1 {pondelok}
+        2 {utorok}
+        3 {stredu}
+        4 {štvrtok}
+        5 {piatok}
+        6 {sobotu}
+        7 {nedeľu}
+        other {{dayNumber} - unknown}
+      }}
+    }',
+	'day-of-week-nth' => '{nth}. {day}',
+	'day-of-week-last' => '{context, select,
+      range {{dayNumber, select,
+        1 {posledného pondelka}
+        2 {posledného utorka}
+        3 {poslednej stredy}
+        4 {posledného štvrtka}
+        5 {posledného piatku}
+        6 {poslednej soboty}
+        7 {poslednej nedele}
+        other {{day} - unknown}
+      }}
+      other {{dayNumber, select,
+        1 {posledný}
+		2 {posledný}
+		3 {poslednú}
+		4 {posledný}
+		5 {posledný}
+		6 {poslednú}
+		7 {poslednú}
+		other {unknown}
+      } {day}}
+    }',
+	'before-day-of-month' => ' ',
+	'day-of-month' => '{day}',
+	'day-of-month-named' => 'dňa v mesiaci {day}',
+	'day-of-month-last-day' => '{context, select,
+      range {posledného dňa v mesiaci}
+      other {posledný deň v mesiaci}
+    }',
+	'day-of-month-last-weekday' => '{context, select,
+      range {posledného pracovného dňa}
+      other {v posledný pracovný deň}
+    }',
+	'day-of-month-nearest-weekday' => '{context, select,
+      range {pracovného dňa najbližšieho k {day}.}
+      other {pracovného dňa najbližšieho k {day}.}
+    }',
+	'before-month' => ' v ',
+	'month' => '{context, select,
+      step {{month, select,
+        1 {januára}
+        2 {februára}
+        3 {marca}
+        4 {apríla}
+        5 {mája}
+        6 {júna}
+        7 {júla}
+        8 {augusta}
+        9 {septembra}
+        10 {októbra}
+        11 {novembra}
+        12 {decembra}
+        other {{month} - unknown}
+      }}
+      range {{month, select,
+        1 {januára}
+        2 {februára}
+        3 {marca}
+        4 {apríla}
+        5 {mája}
+        6 {júna}
+        7 {júla}
+        8 {augusta}
+        9 {septembra}
+        10 {októbra}
+        11 {novembra}
+        12 {decembra}
+        other {{month} - unknown}
+      }}
+      other {{month, select,
+        1 {januári}
+        2 {februári}
+        3 {marci}
+        4 {apríli}
+        5 {máji}
+        6 {júni}
+        7 {júli}
+        8 {auguste}
+        9 {septembri}
+        10 {októbri}
+        11 {novembri}
+        12 {decembri}
+        other {{month} - unknown}
+      }}
+    }',
+	'hour+minute' => '{hourNumeric, select,
+      2 {o}
+      3 {o}
+      4 {o}
+      12 {o}
+      13 {o}
+      14 {o}
+      22 {o}
+      23 {o}
+      other {o}
+    } {hourNumeric}:{minute}',
```

Comment:
You don't have to use select when all values are the same

```suggestion
	'hour+minute' => 'o {hourNumeric}:{minute}',
```